### PR TITLE
Added recipe for cssmin

### LINKED ISSUE #3944: Add recipe for cssmin
It would be nice to have cssmin on conda-forge.

https://github.com/zacharyvoase/cssmin

### PR DESCRIPTION
**Pull request by jenzopr**
If merged, this PR closes #3944

### CODE REVIEW

File: recipes/cssmin/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: v{{ version }}.tar.gz
+  sha256: 0649244b431c8d604d443047c66156cf5b0d0677c6beacbdaf40594ee2fd4ea4
+  url: https://github.com/zacharyvoase/cssmin/archive/v{{ version }}.tar.gz
+
+build:
```
Comment (synapticarbors) [inline recipes/cssmin/meta.yaml]: Looks like a good candidate for `noarch: python`

File: recipes/cssmin/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    - cssmin = cssmin:main
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+
+test:
```
Comment (synapticarbors) [inline recipes/cssmin/meta.yaml]: Also add an `imports` test

File: recipes/cssmin/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  license: BSD
+  dev_url: http://github.com/zacharyvoase/cssmin
+  doc_url: http://github.com/zacharyvoase/cssmin
+  home: http://github.com/zacharyvoase/cssmin
+  summary: A Python port of the YUI CSS compression algorithm.
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - csadorf
```
Comment (synapticarbors) [inline recipes/cssmin/meta.yaml]: @csadorf -- Are you ok being added as a co-maintainer of the `cssmin` recipe on conda-forge?

File: recipes/cssmin/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+  run:
+    - python
+
+test:
+  commands:
+    - cssmin --help
+
+about:
+  license_family: BSD
+  license: BSD
```
Comment (synapticarbors) [inline recipes/cssmin/meta.yaml]: Can you determine which flavor of BSD license this is and then use `BSD-3-Clause`, `BSD-2-Clause` etc, whichever is appropriate.

File: recipes/cssmin/meta.yaml (hunk 3 above)
Comment (csadorf) [reply recipes/cssmin/meta.yaml]: Yes, that's fine with me.

File: recipes/cssmin/meta.yaml (hunk 4 above)
Comment (csadorf) [reply recipes/cssmin/meta.yaml]: I think that the Python package cssmin is actually licensed under the MIT-license, but contains code taken from the "YUI CSS Compressor" software, which is licensed under the BSD 3-Clause license.

See: https://github.com/zacharyvoase/cssmin/blob/master/LICENSE

Comment (jenzopr) [reply recipes/cssmin/meta.yaml]: You think its MIT then? The source code does not explicitly state which parts are from the "YUI CSS Compressor"..

Comment (csadorf) [reply recipes/cssmin/meta.yaml]: I don't consider myself a licensing expert, but as far as I understand it, the BSD-3-Clause allows you to use and relicense the code as long as you keep the original copyright notice with it. So in this case the author took BSD-3-Clause licensed code, added their own code, and relicensed everything under the MIT license. They complied with the BSD license by adding the original copyright notice to the new copyright notice.

This package should be classified as licensed under the MIT license.

See also: https://www.patrick-wied.at/blog/how-to-correctly-use-code-you-didnt-write

File: recipes/cssmin/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+
+test:
+  import:
```
Comment (synapticarbors) [inline recipes/cssmin/meta.yaml]: This should be `imports` not `import`:

https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#python-imports

Comment (csadorf) [reply recipes/cssmin/meta.yaml]: Why did the linter not catch that?

Comment (jenzopr) [reply recipes/cssmin/meta.yaml]: Interesting... thanks!

Comment (synapticarbors) [reply recipes/cssmin/meta.yaml]: Good timing with that question. A PR was opened up in the last 24 hours about that very issue: 

https://github.com/conda-forge/conda-smithy/pull/567

**Pull request merged**